qcodes/tests/drivers/keysight_b1500/b1500_driver_tests/test_b1517a_smu.py

```diff
@@ -37,6 +38,7 @@ def test_snapshot():
     smu.use_high_speed_adc()
     smu.source_config(output_range=VOutputRange.AUTO)
     smu.measure_config(measure_range=IMeasRange.AUTO)
+    smu.measure_config(measure_range=VMeasRange.AUTO)
```

Comment:
general tests should not use deprecated API. deprecated API should have it's own test for which the deprecation warning is explicitly silenced.